src/mmrelay/meshtastic_utils.py

```
@@ -790,9 +801,14 @@ def on_meshtastic_message(packet, interface):
     # Reply handling (Meshtastic -> Matrix)
     # If replyId is present but emoji is not (or not 1), this is a reply
     if replyId and not emoji_flag and interactions["replies"]:
-        longname = get_longname(sender) or str(sender)
-        shortname = get_shortname(sender) or str(sender)
-        orig = get_message_map_by_meshtastic_id(replyId)
+        try:
+            longname = get_longname(sender) or str(sender)
+            shortname = get_shortname(sender) or str(sender)
+            orig = get_message_map_by_meshtastic_id(replyId)
+        except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)